src/angularjs-dropdown-multiselect.js

```diff
@@ -117,6 +117,35 @@ directiveModule.directive('ngDropdownMultiselect', ['$filter', '$document', '$co
                     });
                 }
 
+                function deepEqual(x, y) {
+                  if ((typeof x == "object" && x != null) && (typeof y == "object" && y != null)) {
```

Comment:
Rest of lib conforms to strict `===` comparisons and uses single quotes for strings, should match here.

Also, do you know where this polyfill is from? Lodash's [`_.isEqual()`](https://github.com/lodash/lodash/blob/master/lodash.js#L9470) and the first few google results like [`node-deep-equal`](https://github.com/substack/node-deep-equal) are pretty involved, just want to make sure all our bases are covered.